server/liveblog/blogs/tasks.py

```diff
@@ -36,37 +37,92 @@ def publish_embed(blog_id, api_host=None, theme=None):
     return superdesk.upload.url_for_media(file_id)
 
 
-def delete_embed(blog_id):
+def delete_embed(blog_id, theme=None, output=None):
```

Comment:
Yeah, wouldn't it be clearer and easier to manage with more than one method? And it's not clear in the declaration that you can use the method to delete either the output or the theme, but not both. I mean, you really have to read the code to understand what it does. With separate methods, the behaviour would be clearer.